packages/api-server/api_server/models/lifts.py

```diff
@@ -1,12 +1,13 @@
 from typing import List
 
 from pydantic import BaseModel, Field
+from tortoise.contrib.pydantic.creator import pydantic_model_creator
 
 from . import tortoise_models as ttm
 from .ros_pydantic import rmf_building_map_msgs, rmf_lift_msgs
 
 Lift = rmf_building_map_msgs.Lift
-LiftHealth = ttm.LiftHealth
+LiftHealth = pydantic_model_creator(ttm.LiftHealth)
```

Comment:
Just to check, can this use from .health import BasicHealth like ingestor and fleet?